build.cake

```diff
@@ -315,10 +315,18 @@ Task("PushToMyget")
 
         var packageFiles = new List<FilePath>();
 
-        var nuspecFiles = GetFiles("./artifacts/packages/*.nuspec");
-        foreach (var nuspecFile in nuspecFiles)
+        var projects = new[]
+        {
+            "MongoDB.Bson",
+            "MongoDB.Driver.Core",
+            "MongoDB.Driver",
+            "MongoDB.Driver.GridFS",
+            "mongocsharpdriver" // the Nuget package name for MongoDB.Driver.Legacy
+        };
```

Comment:
I prefer to list the projects explicitly than to glob for them, because not all projects correspond to a package that needs to be uploaded.